modules/dashboard/cloudevent-receiver/variables.tf

```diff
@@ -12,3 +12,15 @@ variable "triggers" {
   description = "A mapping from a descriptive name to a subscription name prefix."
   type        = map(string)
 }
+
+variable "alert_threshold" {
+  description = "The threshold for the alerting policy."
+  type        = number
+  default     = 50000
```

Comment:
This should really be per type, could we add it to the value of the triggers map?

Reply:
I think `triggers` should be:
```hcl
map(object({
  name string
  alert_threshold optional(number)
}))
```

This will be breaking

Reply:
Done